### PR TITLE
Add a section about collecting logs through HTTP

### PR DESCRIPTION
**Pull request by sebartyr**
## Describe your PR

This PR suggests to add a section in log management about creating an http drain to collect logs, hosted on Clever Cloud.

## Checklist

- [ ] My PR is related to an opened issue : #
- [ ] I've read the [contributing guidelines](../CONTRIBUTING.md)

## Reviewes
_Who should review these changes?_ @juliamrch 

_Commit checked with vale and markdownlint_

### CODE REVIEW

**Comment by sebartyr** [comment]
Feel free to close this PR if it is not relevant

File: content/doc/administrate/log-management.md [hunk 1]
```diff
@@ -198,6 +198,14 @@ curl -X PUT "https://username:password@xxx-elasticsearch.services.clever-cloud.c
 
 For more information, please refer to the [official documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/set-up-lifecycle-policy.html).
 
+### HTTPS
+
+There is a side project, available on GitHub which enables to create a drain to collect logs through an HTTPS endpoint. It is fully compatible with Clever Cloud.
```
Comment (juliamrch) [inline content/doc/administrate/log-management.md:203]:
```suggestion
Some tools available on GitHub enable to create a drain to collect logs through an HTTPS endpoint. [This project](https://github.com/sebartyr/http-logs-drain), for example, is fully compatible with Clever Cloud 
```

File: content/doc/administrate/log-management.md [hunk 2]
```diff
@@ -198,6 +198,14 @@ curl -X PUT "https://username:password@xxx-elasticsearch.services.clever-cloud.c
...
+
+You could host it as an app and an add-on on Clever Cloud. A complete README explains all the features.
+
+You can find the repository of the project [here](https://github.com/sebartyr/http-logs-drain).
```
Comment (juliamrch) [inline content/doc/administrate/log-management.md:207]:
```suggestion
```

**Review by juliamrch — CHANGES_REQUESTED** [review]
Noice. Added suggestions for a more neutral tone.

**Comment by sebartyr** [comment]
Thanks for your comments. I commited the changes

**Review by juliamrch — APPROVED** [review]
noice

**Pull request merged**